SubGHz External High Power

Z3BRO opened the pull request:
# What's new

Adds support for TehRabbitt's Flux Capacitor Board

Tested with my Flux Capacitor board, works great

# Verification 

Build with ` ./fbt COMPACT=1 DEBUG=0 updater_package` and test with Flux Capacitor Board

# Checklist (For Reviewer)

- [x] PR has description of feature/bug
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

Z3BRO committed:
SubGHz External High Power

beckerj commented:
Tested with several external C1101 module (Ebyte, fake Ebyte, C1101 900Mhz TehRabbbit with E07-900MM10S package and internal C1101) - no problems at all.

RogueMaster commented:
Several RogueMaster users have tested this code on the RogueMaster firmware and it works as intended

xMasterX committed:
Remake it more properly

still has problems, need to rewrite subghz_devices tx/rx switch code, checking strcmp every time is not good too, but better than loading whole config from microsd

xMasterX reviewed (approved):
Hi
I did some changes to this code, see last commit
Still requires some changes, that might be done later in dev branch
Also you did not said anything about this code authors, I will do it in changelog
